src/test/java/com/google/cloud/tools/gradle/appengine/standard/AppEngineStandardExtensionTest.java

```diff
@@ -66,6 +67,18 @@ private Project setUpTestProject(String buildFileName) throws IOException {
     return p;
   }
 
+  // TODO : Make this a test that reads ALL params
```

Comment:
So right now it's only reading in the environment and checking that it is set correctly. Potentially we could read in every parameter and check if we're setting and getting correctly.